api/src/Entity/Customer.php

```diff
@@ -78,4 +95,20 @@ public function setAvatarUrl(string $avatarUrl): void
     {
         $this->avatarUrl = $avatarUrl;
     }
+
+    /**
+     * @return Collections\ArrayCollection
+     */
+    public function getLunchTrains(): Collections\ArrayCollection
+    {
+        return $this->lunchTrains;
+    }
+
+    /**
+     * @param Collections\ArrayCollection<LunchTrain> $lunchTrains
+     */
+    public function setLunchTrains(Collections\ArrayCollection $lunchTrains): void
```

Comment:
Am I right in assuming that we need to expose collections here?

Reply:
yes